web/themes/custom/lex/patterns/components/header.twig

```diff
@@ -17,7 +17,6 @@
             <span class="hamburger-line"></span>
             <span class="hamburger-line"></span>
           </div>
-          {# <span class="menu-label">Search</span> #}
```

Comment:
Was this a requirement of google translate. Or is this just tweaks during iterating on the multidev environment?

Reply:
They are commented out lines. I just removed them to remove them.